### PR TITLE
Fix MNTOR-1795 - Add ADR for L10N workflow updates

### PR DESCRIPTION
**Pull request by maxxcrawford**
## Summary

This ADR captures a decision to refactor how L10N works on Monitor. 

Links:
- [MNTOR-1795](https://mozilla-hub.atlassian.net/browse/MNTOR-1795)

### CODE REVIEW

File: docs/adr/0006-localization-workflow.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# L10N Workflow
+
+* Status: proposed
```
Comment (maxxcrawford) [inline docs/adr/0006-localization-workflow.md:3]: Update status once approved

**Review by flodolo — CHANGES_REQUESTED** [review]

File: docs/adr/0006-localization-workflow.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+### Seperate Repo / Semi-automated Syncing 
+
+The proposal is to use a separate l10n repository instead of a separate branch, then use automation to sync only the relevant files between the two repositories instead of a git submodule.
+
+*For additional sentistive considerations, see [this document](https://docs.google.com/document/d/12dRW85DNZpljmoS-U9KrZqV1nBDrT2g8WTpvTAM0HgY/edit?usp=sharing)* 
+
+* Good, there are no changes for development compared to the existing workflow:
+* Good, This workflow doesn’t block developers from landing code in the main branch as part of code pull requests.
+* Good, It also allows queuing multiple changes before exposing them for localization, since strings are exposed through a pull request, which is manually reviewed by the localization EPM.
+* Good, localizers workflow, new strings will trigger localization review requests via CODEOWNERS.
```
[the remark this answers is not in the document]
Comment (maxxcrawford) [reply docs/adr/0006-localization-workflow.md]: Removed in 13e494d2a9ff8c05f30ace880a3dbe5256d46e0d

File: docs/adr/0006-localization-workflow.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Technical Story: [MNTOR-1795](https://mozilla-hub.atlassian.net/browse/MNTOR-1795)
+
+## Context and Problem Statement
+
+The current L10N workflow is a complex, multiple manual-step process to sync strings between two branches. This process is error-prone. Because of its lack of automation, it also often does not give the L10N team enough time to localize strings. 
+
+## Decision Drivers 
+
+* With the React refactor, this is a good time to revisit our processes, and determine what makes the most since
```
[the remark this answers is not in the document]
Comment (maxxcrawford) [reply docs/adr/0006-localization-workflow.md]: Rewritten in 13e494d2a9ff8c05f30ace880a3dbe5256d46e0d

> With the React refactor, this is a good time to revisit our processes, and determine what works the best for everyone.

File: docs/adr/0006-localization-workflow.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Technical Story: [MNTOR-1795](https://mozilla-hub.atlassian.net/browse/MNTOR-1795)
+
+## Context and Problem Statement
+
+The current L10N workflow is a complex, multiple manual-step process to sync strings between two branches. This process is error-prone. Because of its lack of automation, it also often does not give the L10N team enough time to localize strings. 
+
+## Decision Drivers 
+
+* With the React refactor, this is a good time to revisit our processes, and determine what works the best for everyone.
+  * This is also a chance to better align with other Mozilla projects
```
Comment (rhelmer) [inline docs/adr/0006-localization-workflow.md:16]:
```suggestion
  * This is also a chance to better align with other Mozilla projects.
```

**Review by rhelmer — APPROVED** [review]
lgtm, flod is the expert here so I'll defer to him. Thanks for writing this up!

File: docs/adr/0006-localization-workflow.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+## Considered Options
+
+* Separate Repo / git Submodules (Relay)
+* Seperate Repo / Automated Syncing
```
Comment (flodolo) [inline docs/adr/0006-localization-workflow.md]: All the other `seperate` typos are still around.

**Review by flodolo — APPROVED** [review]
Fixed some of the nits I reported in a separate commit, hope that's OK.

**Pull request merged**